test/test_scalar_hoisting.cpp

```diff
@@ -300,33 +300,33 @@ TEST_F(ScalarHoistTest, IndexHoist3) {
   const std::string expected_kernel = R"(
 __global__ void CUDAGeneratedKernel(Tensor<float, 2, 2> T0, Tensor<float, 2, 2> T2) {
   nvfuser_index_t i0;
-  i0 = ((nvfuser_index_t)threadIdx.x) + (256 * ((nvfuser_index_t)blockIdx.x));
+  i0 = ((nvfuser_index_t)threadIdx.x) + (256LL * ((nvfuser_index_t)blockIdx.x));
```

Comment:
We'd better get rid of this. It tests implementation not behavior...

Reply:
The intention here is to verify hoisting of a scalar is indeed performed, so the literal matching is sufficient but not necessary. Some more intelligent matching would reduce false alarms, but we shouldn't just remove this check.

Reply:
Sure. I'll leave this test case as is, but hope someone will fix it later.